src/Admin/ReaderThemes.php

```diff
@@ -223,7 +223,7 @@ public function get_default_reader_themes() {
 				'query_themes',
 				[
 					'author'   => 'wordpressdotorg',
-					'per_page' => 24, // There are only 12 as of 05/2020.
+					'per_page' => 24, // There are only 13 as of 12/2020.
```

Comment:
FYI: The `Test_Reader_Theme_REST_Controller::test_get_items()` test will begin to fail once `twentytwentyone` is published as a theme.

Reply:
I don't think it will because it's checking if the list is equal, but rather that `$expected_reader_themes` is a subset of `$actual_reader_themes`.

Reply:
Ah right, my bad. We should add `twentytwentyone` to `$expected_reader_themes` when the theme is published, nevertheless.